templates/user/settings/repos.tmpl

```diff
@@ -86,7 +86,7 @@
 							<div class="item">
 								<div class="content flex-text-block">
 									{{if .IsPrivate}}
-										{{svg "octicon-lock" 16 "text gold"}}
+										{{svg "octicon-lock"}}
```

Comment:
Why it's not `octicon-repo-locked`

Reply:
I think this part of the template is broken, at least I can not get this condition to render. When I circumvent the `{{if or .allowAdopt .allowDelete}}` condition,  `.Repos` is empty. I will fix it, but I think it may be dead template code.

Reply:
This is fixed now.